libcst/matchers/_matcher_base.py

```diff
@@ -77,18 +80,45 @@ def __invert__(self: _BaseMatcherNodeSelfT) -> "_BaseMatcherNodeSelfT":
 def DoNotCare() -> DoNotCareSentinel:
     """
     Used when you want to match exactly one node, but you do not care what node it is.
-    Useful inside sequences such as function def and argument matchers. You do not
-    need to use this for attributes since :func:`DoNotCare` is already the default.
+    Useful inside sequences such as a :class:`libcst.matchers.Call`'s args attribte.
```

Comment:
The libcst.matchers.Call link doesn't work. https://2645-200896124-gh.circle-artifacts.com/0/doc/matchers.html
(When it works, it's clickable and shows only the simple class name Call.)
![image](https://user-images.githubusercontent.com/3840867/66003476-0b0bf280-e45b-11e9-8611-b3c29f9b8e31.png)

Reply:
I can change that back, but this is intentional since we don't have a page rendering out concrete classes. They're identical to the LibCST classes so I don't think adding them gives us any benefit.